test/integration/smoke/test_volumes.py

```diff
@@ -858,7 +858,7 @@ def test_10_list_volumes(self):
 
     def wait_for_attributes_and_return_root_vol(self):
 
-        for i in range(60):
+        for i in range(360):
```

Comment:
@borisstoyanov can you rewrite the test using wait_until instead, for example:
```
# If not import do this:
from marvin.lib.utils import wait_until
[...snipped...]

     def wait_for_attributes_and_return_root_vol(self):
        def checkVolumeResponse():
               list_volume_response = Volume.list(
                  self.apiClient,
                  virtualmachineid=self.virtual_machine.id,
                 type='ROOT',
                 listall=True
             )
             if  list_volume_response[0].virtualsize is not None:
                 return True, list_volume_response[0]
             return False, None

        # sleep interval is 1s, retries is 360, this will sleep atmost 360 seconds, or 6 mins
        res, response = wait_until(1, 360, checkVolumeResponse)                                                                      
        if not res:                                                                
            self.fail("Failed to return root volume response")                 
        return response 
```

Reply:
done